R/assocTestSingle.R

```diff
@@ -4,7 +4,8 @@ setGeneric("assocTestSingle", function(gdsobj, ...) standardGeneric("assocTestSi
 ## do we want to make imputing to the mean optional?
 setMethod("assocTestSingle",
           "SeqVarIterator",
-          function(gdsobj, null.model, test=c("Score", "Wald"), GxE=NULL, sparse=TRUE, imputed=FALSE, male.diploid=TRUE, genome.build=c("hg19", "hg38"), verbose=TRUE) {
+          function(gdsobj, null.model, test=c("Score", "Score.SPA"), recalc.pval.thresh=1, GxE=NULL,
```

Comment:
Should we make the default for `recalc.pval.thresh` 0.05?  Adrienne's results showed that for variants with p > 0.05, there was pretty much no difference at all.